### DISCUSSION TITLE
ODE notation

### DISCUSSION THREAD

**Discussion by pnkraemer** [probabilistic-numerics/probnum#485] [Research]
Do we use `dx = f(x)` or do we use `dy = f(y)` to write ODEs? Seems to me like we use both about 50 percent of the time: in the ode module, we use y, in the odesolvers, we use x.

I dont have an opinion as to what is better, but dont like that we use both

**Comment by pnkraemer** [comment]
in case we want to decide on something: if we start thinking about PDEs, neither x nor y make a lot of sense. Instead, PDEs often use "u" (for "unknown") which is notation that would apply to both. Then, however, do we want the same letter for both?

**Comment by nathanaelbosch** [comment]
When I write things down I tend to use `dy = f(y)`, and `X` for the states. At the same time I'm also very used to `u` in code since that's what they use in DifferentialEquations.jl, so I would also be fine with using `u` everywhere (and at that point, I might even start using it when I write about it ;))
In any case, very good point, we should definitely adjust the documentation to be consistent.

**Comment by pnkraemer** [comment]
I think I prefer "y" (even though I tend to write "x" in my notes). The reasons are that (1l contra "x") "x" is omnipresent, so it will at some point lead to confusion; (2; contra "u") I think it is good to not use the same letter as for e.g. PDEs, for which the method of lines translates a PDE into an ODE which can (in theory at least) tackled with any ODE solver and I like that this may be reflected in notation.

**Comment by pnkraemer** [comment]
On a related note (which came up in the most recent tutorial PR #292 ), let's establish a consistent terminology regarding Wiener processes/Brownian motion and the respective notation.
